src/components/VHeader/VHeaderMenu.vue

```diff
@@ -57,14 +58,12 @@ export default defineComponent({
       })
       router.push(newPath)
 
-      function typeWithoutMedia(mediaType) {
-        return mediaStore.resultCountsPerMediaType[mediaType] === 0
-      }
-
       const shouldFetchMedia =
         type === ALL_MEDIA
```

Comment:
This is an interesting one (on line 62):

> (parameter) type: "image" | "audio"
This condition will always return 'false' since the types '"image" | "audio"' and '"all"' have no overlap.